config/module.config.php

```diff
@@ -292,3 +289,9 @@
         ],
     ],
 ];
+
+if (class_exists(ImportCommand::class)) {
+    $result['service_manager']['invokables']['doctrine.dbal_cmd.import'] = ImportCommand::class;
+}
+
+return $result;
```

Comment:
Why are we not promoting `Doctrine\DBAL\Tools\Console\Command\ReservedWordsCommand` here as well? <br>Available with [2.13](https://github.com/doctrine/dbal/blob/2.13.x/lib/Doctrine/DBAL/Tools/Console/Command/ReservedWordsCommand.php) and [3.1](https://github.com/doctrine/dbal/blob/3.1.x/src/Tools/Console/Command/ReservedWordsCommand.php)